jsk_perception/CMakeLists.txt

```diff
@@ -234,6 +234,7 @@ jsk_add_nodelet(src/sparse_image_decoder.cpp "jsk_perception/SparseImageDecoder"
 jsk_add_nodelet(src/color_histogram.cpp "jsk_perception/ColorHistogram" "color_histogram")
 jsk_add_nodelet(src/background_substraction_nodelet.cpp "jsk_perception/BackgroundSubstraction" "background_substraction")
 jsk_add_nodelet(src/fisheye_to_panorama.cpp "jsk_perception/FisheyeToPanorama" "fisheye_to_panorama")
+jsk_add_nodelet(src/point_pose_extractor.cpp "jsk_perception/PointPoseExtractor" "point_pose_extractor")
```

Comment:
Could you add test?
https://github.com/jsk-ros-pkg/jsk_recognition/blob/0de1e9ac4ec9ea27c548d835c9ca5e94a6101808/jsk_perception/CMakeLists.txt#L468-L582

Reply:
I added the test in 4a7aeda

Reply:
Nice!